readthedocs/search/api.py

```diff
@@ -62,7 +62,7 @@ def get_queryset(self):
         # Validate all the required params are there
         self.validate_query_params()
         query = self.request.query_params.get('q', '')
-        kwargs = {}
+        kwargs = {'filter_user': False}
```

Comment:
nitpick: I'd like to come up with a better name for this. I'm thinking about `filter_by_user` which is a little more explicit.

What we want to communicate here is "filter versions by users permissions" I suppose, but I didn't find a good name for that :(